src/controllers/shoppingCartController/viewing.js

```diff
@@ -0,0 +1,19 @@
+import Stripe from 'stripe';
+import db from '../../database/models';
+import { asyncWrapper } from '../../utils/handlingTryCatchBlocks';
+import { Op } from 'sequelize';
+
+export const viewOrderDetails = asyncWrapper(async(req,res) =>{
+  const trackingnumber = req.params.trackingId;
+  const order = await db.Order.findOne({
+    where: {
+        trackingNumber : trackingnumber
+    }
+  })
+  const details = await db.OrderDetails.findAll({
+    where: { id: { [Op.in]: order.products } },
+  });
+  res.status(200).json({data: details})
```

Comment:
can you include the status and message properties within response

Reply:
check again

Reply:
DONE